### DISCUSSION TITLE
Public contributions to an organization with SSO show incorrectly

### DISCUSSION THREAD

**Discussion by MarshalX** [community/community#81766] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

**Description:** public contributions to an organization with SSO don't show **to me** without login

<img width="783" alt="image" src="https://github.com/community/community/assets/15520314/09b99e54-b6fb-45f1-9859-6f5fe94cce8a">

but if I open my profile in the private tab it will be visible

<img width="780" alt="image" src="https://github.com/community/community/assets/15520314/505cc177-8634-428e-89dc-379c057bbd68">

I can see it under my profile after passing SSO only. The contributions that are not visible to me are from **public** org repositories.

**Actual:** public contributions from org with SSO aren't visible to me
**Expected:** public contributions from org with SSO will be visible for me like it is visible to others

**Comment by Anixii** [comment]
a
